src/Components/Wizard/types.ts

```diff
@@ -5,11 +5,23 @@ export type FormErrors = { [key: string]: any } | null
 
 export type StepElement = React.ReactElement<StepProps>
 
+// Extending WizardContext here causes
+// `Type '(e: FormEvent<HTMLFormElement>) => any' is not assignable to type '(event: MouseEvent<Button>) => void'
+// in form.handleSubmit for button onClick prop
+// OR '(e: FormEvent<any>, values: { [key: string]: any; }) => void' is not assignable to type '(event: MouseEvent<Button>) => void'
+// for wizard.previous  for button OnClick prop
+export interface WizardStepChildProps {
+  form: any
+  wizard: any
+}
```

Comment:
The props passed via context to a Wizard's Step's Child somehow cause type errors as descibed above, basically a conflict between MouseEvents and FormEvents. This isn't a problem in the wizard example form, where we don't use context from Step, so I'm solving it by setting these props to `any`. This avoids needing to say `< ... onClick={ form.handleSubmit as any }>` in the shipping, review and payment forms.